tests/Behat/Context/Transform/PriceContext.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RefundPlugin\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+
+final class PriceContext implements Context
+{
+    /**
+     * @Transform /^"(\-)?((?:\d+\.)?\d+)"$/
```

Comment:
What's the reason behind matching quotation marks, not just the value inside them?

Reply:
I didn't change the implementation from `LexicalContext` from Sylius, but I guess that it had been written that way to this transformer doesn't catch some unwanted strings (as order number in scenarios from this plugin)